bin/varnishd/cache/cache_backend_probe.c

```diff
@@ -181,7 +181,7 @@ VBP_Update_Backend(struct vbp_target *vt)
 	vt->backend->sick = i;
 
 	AN(dir->vcl_name);
-	VSL(SLT_Backend_health, 0, "%s %s %s %s %u %u %u %.6f %.6f %s",
+	VSL(SLT_Backend_health, 0, "%s %s %s %s %u %u %u %.6f %.6f \"%s\"",
```

Comment:
we should probably mark here with an XXX that the `%s` must not contain any quotes

Reply:
This is something we get from the backend, if the reason of a backend response contains double quotes there's little I'm willing to do to preserve the last field of `Backend_health`. The reason is only meant to be human-readable so as long as we have the protocol version and status this is fine.

Please note that there is no `:reason` pseudo header field since h2.